src/cozmo/conn.py

```diff
@@ -376,11 +376,19 @@ def _handle_ui_device_connected(self, msg):
 
         self._is_ui_connected = True
         self.dispatch_event(EvtConnected, conn=self)
-        self.anim_names.refresh()
+
         logger.info('App connection established. sdk_version=%s '
                 'cozmoclad_version=%s app_build_version=%s',
                 version.__version__, cozmoclad.__version__, msg.buildVersion)
 
+        # We RequestAvailableObjects before the animation names as this ensures
```

Comment:
Instead of "We RequestAvailableObjects before the animation names", how about "We call RequestAvailableObjects before refreshing the animation names"?